infra/gcp/bash/lib_iam.sh

```diff
@@ -215,6 +215,26 @@ function ensure_secret_role_binding() {
     _ensure_resource_role_binding "secrets" "${secret}" "${principal}" "${role}"
 }
 
+# Ensure that IAM binding is present for repositories
+# Arguments:
+#   $1:  The repository name (e.g. "images")
+#   $2:  The principal (e.g. "group:k8s-infra-foo@kubernetes.io")
+#   $3:  The role name (e.g. "roles/storage.objectAdmin")
+#   $4:  The project (e.g. "k8s-artifacts-prod")
+#   $5:  The location (e.g. "europe")
+function ensure_repository_role_binding() {
```

Comment:
```suggestion
function ensure_ar_repository_role_binding() {
```

We should also move this function ot `lib_ar.sh` since it's only scoped to artifact registry repositories.

Reply:
Fixed.